README.md

```diff
@@ -28,10 +28,12 @@ Configures a PWM port on the EdgePi.
 
 Example input configuration:
 ```
-msg.payload: 0
-msg.frequency: 1000
-msg.polarity: NORMAL
-msg.enabled: false
+msg {
+  "payload": "0",
+  "frequency": "1000",
+  "polarity": "NORMAL",
+  "enabled": "false"
+}
```

Comment:
are we expecting everything as string or this is  a typo?

Reply:
typo. I thought everything in json had quotes. fixed.